tests/solana.rs

```diff
@@ -80,20 +78,18 @@ type CallParametersCheck = fn(vm: &VirtualMachine, instr: &Instruction, pda: &[P
 
 struct VirtualMachine {
     account_data: HashMap<Account, AccountState>,
-    origin: Account,
-    programs: Vec<Contract>,
-    stack: Vec<Contract>,
+    programs: Vec<Program>,
+    stack: Vec<Program>,
     logs: String,
     events: Vec<Vec<Vec<u8>>>,
     return_data: Option<(Account, Vec<u8>)>,
     call_params_check: HashMap<Pubkey, CallParametersCheck>,
 }
 
 #[derive(Clone)]
-struct Contract {
-    program: Account,
+struct Program {
+    id: Account,
```

Comment:
I am not keen on `id`. I don't like `program_id` either, I think it should have been called `program_account` or so. But we're stuck with it, so I guess we just have to live with `id`. :cry:

Reply:
My inspiration was `program_id`, but I later noticed the code does not look so cool with `id` either. Do you have any other suggestion for naming this field?

Reply:
No, I don't have any ideas. I just don't like it very much - pointless comment